monitoring/src/check-page/common-functions.ts

```diff
@@ -47,7 +47,12 @@ export const ScreenDimensions = {
  * @return {*}  {Promise<void>}
  */
 export const clearCookies = async (page: Page): Promise<void> => {
-	await page.context().clearCookies();
+	const cookies = await page.context().cookies();
+
+	for(const cookie of cookies) {
+		await page.context().clearCookies(cookie);
```

Comment:
How is this better than `clearCookies()`? Could maybe do with a comment for future travellers so nobody ends up "optimizing" this and reverting your change!